roadmap/implementers-guide/src/runtime/dmp.md

```diff
@@ -0,0 +1,57 @@
+# DMP Module
```

Comment:
My logic the users of this document should use accronyms extensibly. To the point that we never expand HRMP, we just use it to name everything related to it.

Secondly, I am really not a fan of duplicating information, because in the definition is given already in the glossary and perhaps the messaging overview.

Reply:
I guess it's a philosophical difference: I like to document things everywhere, because it's hard to predict the directions from which users will come. HRMP is a great example of that, actually: internally, I always expand it to "higher-ranked message passing", which I am fairly confident is incorrect, but we're just using the acronym everywhere. Most of the time that's fine, in usage, but it seems like a fairly tame redundancy to also define it in its section header in the guide.

Reply:
Ok fair, I am gonna expand these acronyms however only in the text below. The reasoning here is that the title is the module name, and that is the abbreviation.